source/crud/tests/unified/aggregate-let.yml

```diff
@@ -115,3 +122,52 @@ tests:
                 aggregate: *collection0Name
                 pipeline: *pipeline1
                 let: *let1
+
+  - description: "Aggregate to collection with let option"
+    runOnRequirements:
+      - minServerVersion: "5.0"
+    operations:
+      - name: aggregate
+        object: *collection0
+        arguments:
+          pipeline: &pipeline2
+            - $match: { $expr: { $eq: ["$_id", "$$id"] } }
+            - $project: { _id: 1 }
+            - $out: *collection1Name
```

Comment:
@jyemin IIRC, you originally hit the error from [SERVER-57403](https://jira.mongodb.org/browse/SERVER-57403) using `$merge`. Is it at all relevant that this test is using `$out` instead?

I'm also curious because [SERVER-46998](https://jira.mongodb.org/browse/SERVER-46998) doesn't mention `$out` (only `$merge`). Perhaps this doesn't matter at all since the purpose of this test is to exercise the _driver's_ code path(s).

Reply:
Yes, that's right.  I'm just trying to exercise the driver here.